docs/dashboards/chart-builder.md

```diff
@@ -176,3 +190,18 @@ Variables can be created in Report options "Manage variables".
 
 ![Manage Variables](../images/dashboards/variables.png)
 
+
+## Style
+
+In the style tab, you configure what the chart looks like. You can select bar, line, area and point 
+charts. In addition to chart type, you can choose a color from the palette and group color scheme. The main
+goal of a group color scheme is to determine the colors of the series from the same group. When you group metric by
```

Comment:
goal of a group color scheme is to determine the colors of the series from the same group. When you group metric by
=>
goal of a group color scheme is to determine the colors of the series from the same group. When you group metrics by

Reply:
I intentionally used `metric` because you group a single metric by a tag

Reply:
Aha! Then it should be "a metric". Add an "a" before `metric` and you're good to go. Merge away. :smile: